tests/config/test_config.py

```diff
@@ -120,3 +120,25 @@ def test_inconsistent_dropout_use(test_config_filename):
         match="To use dropout timedeltas dropout fraction should be > 0",
     ):
         _ = Configuration(**configuration.model_dump())
+
+
+def test_accum_channels_validation(test_config_filename):
+    """Test accum_channels validation with required normalization constants."""
+    # Load valid config (implicitly tests valid case)
+    config = load_yaml_configuration(test_config_filename)
+    nwp_name, _ = next(iter(config.input_data.nwp.root.items()))
+
+    # Test invalid channel scenario
+    invalid_config = config.model_copy(deep=True)
+    invalid_nwp = invalid_config.input_data.nwp.root[nwp_name]
+    invalid_nwp.accum_channels = ["invalid_channel"]
+
+    # Verify exact error message
+    expected_error = (
+        r"input_data.nwp.ukv\n"
```

Comment:
```suggestion
        rf"input_data.nwp.{nwp_name}\n"
```
Will something like this work? It seems strange to hardcode the source when it's not hardcoded in the test itself (which in my opinion is the way to do it btw - we do hardcode it in other places, but I think this is better practice)